src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Product.orm.xml

```diff
@@ -62,6 +62,7 @@
 
         <one-to-many field="images" target-entity="Sylius\Component\Core\Model\ProductImageInterface" mapped-by="owner" orphan-removal="true">
             <order-by>
+                <order-by-field name="position" />
                 <order-by-field name="id" />
```

Comment:
If we use gedmo to sort position, there won't be any duplicate positions, so this line is unnecessary.